usr/sbin/p11sak/p11sak.c

```diff
@@ -26,6 +28,54 @@ static const char *default_pkcs11lib = "libopencryptoki.so";
 static void *pkcs11lib = NULL;
 static CK_FUNCTION_LIST *funcs = NULL;
 
+static void error_hook(int line, int col, const char *msg)
+{
+  fprintf(stderr, "PARSE ERROR: %d:%d: %s\n", line, col, msg);
+}
+
+void dump_attr(CK_ATTRIBUTE_PTR a)
+{
+    const char *typestr;
+
+    typestr = p11_get_cka(a->type);
+
+    switch (a->ulValueLen) {
+    case 0:
+        printf("  %s: len=0 pValue=%p\n", typestr, a->pValue);
```

Comment:
`a->pValue` should be `NULL` in this case

Reply:
Done